hook.js

```diff
@@ -62,17 +62,14 @@ function getConfig(repo, branch) {
     var configFilename = '.doiuse';
```

Comment:
Should we move this to the top of the file?  It's a constant, maybe better to put it up there